CHANGELOG.md

```diff
@@ -2,10 +2,78 @@
 
 ## Release 2.4.4 (in development)
 
+### Features added
+- [#2561](https://github.com/scality/metalk8s/issues/2561) - install `kubectl`
+on all master nodes (PR [#2562](https://github.com/scality/metalk8s/pull/2562))
+
+### Security fixes
+- Due to critical vulnerabilities (
+[CVE-2020-11652](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11652)
+and
+[CVE-2020-11651](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11651))
+with CVSS score of 10.0 that was discovered affecting all `Salt Master`
+versions inferior to `3000.2`, this release ships with a `Salt Master` version
+updated to `3000.3`. Users, especially those who expose the `Salt Master` to
+the internet must therefore upgrade immediately.
+
+    [#650](https://github.com/scality/metalk8s/issues/650) - Upgrade Salt master
+    to version `3000.3`
+    (PR [#2549](https://github.com/scality/metalk8s/pull/2549))
+
+- Due to an incorrect access control vulnerability
+[CVE-2020-13379](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13379)
+with CVSS score of 5.3 that was discovered affecting Grafana versions from
+3.0.1 through 7.0.1, this release ships with a Grafana version updated to
+`6.7.4`. For more, see [here](https://grafana.com/blog/2020/06/03/grafana-6.7.4-and-7.0.2-released-with-important-security-fix/)
+
+    [#2600](https://github.com/scality/metalk8s/issues/2600) - Upgrade Grafana
+    to `6.7.4` (PR [#2605](https://github.com/scality/metalk8s/issues/2605))
+
+### Enhancements
+- Make `etcd` expansions more resilient
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
+
+- Add documentation on how to downgrade Salt
+(PR [#2540](https://github.com/scality/metalk8s/pull/2540))
+
+- [#2589](https://github.com/scality/metalk8s/issues/2589) - Bump Kubernetes
+version to 1.15.12 (PR [#2595](https://github.com/scality/metalk8s/pull/2595))
+
+- [#2029](https://github.com/scality/metalk8s/issues/2029) - Bump
+python-kubernetes client to `v11`
+(PR [#2554](https://github.com/scality/metalk8s/pull/2554))
```

Comment:
Minor: To me, we should put the bump version as the beginning of the enhancements (or give it a specific section)

Reply:
Moved to top of enhancements!